macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -112,6 +112,10 @@
     {{ elementary.file_log("Inserting metrics into {}.".format(target_relation)) }}
     {%- do elementary.run_query(dbt.create_table_as(True, temp_relation, test_tables_union_query)) %}
     {% do elementary.run_query(insert_query) %}
+
+    {% if not elementary.has_temp_table_support() %}
+        {% do adapter.drop_relation(temp_relation) %}
+    {% endif %}
```

Comment:
This will ensure temporary tables are drop, if an adapter, like Trino, does not support them.